src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -12,4 +12,8 @@
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
 
+    public static final Prefix PREFIX_DEADLINE = new Prefix("d/");
+    public static final Prefix PREFIX_PRIORITY = new Prefix("p/");
```

Comment:
Priority and phone has the same prefix, may need to be careful later

Reply:
Ok thanks!